docs/UserGuide.md

```diff
@@ -282,7 +282,7 @@ Examples:
 
 Finds one or more patients whose tag exactly match the given keyword(s).
 
-Format: `findt KEYWORD [MORE_KEYWORDS]`
+Format: `findt KEYWORD [MORE_KEYWORDS]…​`
```

Comment:
Would [MORE_KEYWORD] be better than [MORE_KEYWORDS] here?

Reply:
Yes i agree!